src/Flex.php

```diff
@@ -594,6 +594,18 @@ private function fetchRecipes(): array
             }
 
             if (isset($manifests[$name])) {
+                if (!empty($manifests[$name]['manifest']['conflict'])) {
+                    $lockedRepository = $this->composer->getLocker()->getLockedRepository();
+
+                    foreach ($manifests[$name]['manifest']['conflict'] as $conflictingPackageName => $constraint) {
+                        if ($conflictingPackage = $lockedRepository->findPackage($conflictingPackageName, $constraint)) {
+                            $this->io->writeError(sprintf('    <warning>Recipe for %s conflicts with %s:%s which should not be %s</>', $name, $conflictingPackageName, $conflictingPackage->getPrettyVersion(), $constraint));
+                            $this->lock->remove($name);
```

Comment:
A conflict should not change the state of the lock file.

Reply:
I guess that a package should not be added to the lock if it's not configured because of a conflict